homeassistant/components/tessie/config_flow.py

```diff
@@ -38,6 +38,7 @@ async def async_step_user(
         """Get configuration from the user."""
         errors: dict[str, str] = {}
         if user_input:
+            self._async_abort_entries_match(dict(user_input))
```

Comment:
Please add a config flow tests that aborts because there is already a config entry with the token.

Reply:
Done